pkg/types/types.go

```diff
@@ -69,6 +71,7 @@ type Result struct {
 	IP6 *IPConfig `json:"ip6,omitempty"`
 }
 
+// Print is a tool for specific use
```

Comment:
`Print` is for outputting a result to stdout -- the protocol for plugins is that they are given arguments in the process environment, and output the result on stdout.